apps/vercel/frontend/src/components/common/GrayInfoBox/GrayInfoBox.styles.ts

```diff
@@ -1,7 +1,8 @@
 import tokens from '@contentful/f36-tokens';
 import { css } from 'emotion';
 
-export const useStyles = (rootOptions?: {}) => ({
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const useStyles = (rootOptions: any) => ({
```

Comment:
this was a really hard type to figure out and I time-boxed it since I don't think its imperative, though would be nice to have.